site/docs/concepts/config-policies.md

```diff
@@ -188,3 +188,29 @@ cd fybrik
 helm install fybrik-crd charts/fybrik-crd -n fybrik-system --wait
 helm install fybrik charts/fybrik --set global.tag=master --set global.imagePullPolicy=Always -n fybrik-system --wait
 ```
+
+### How to add start and/or expiry dates to policies
+
+By utilizing the time built-in functions of OPA, each policy can be added with an expiry date. The related built-in functions are:
+
+``` 
+output := time.now_ns() //Output is a number representing the current time since epoch(1970.1.1) in nanoseconds, in the time zone of UTC.
```

Comment:
Shorten the comment: // get the current time

Reply:
Changed